lib/PanelTestProvider.jsm

```diff
@@ -240,6 +240,110 @@ const MESSAGES = () => [
       patterns: ["*://*/*.pdf"],
     },
   },
+  {
+    id: "SEND_TAB_CFR",
```

Comment:
Maybe something about recipes specifically since there's likely to be multiple variants of this? I guess this is for testing anyway so it doesn't really matter